sdk/docs/iot/README.md

```diff
@@ -54,9 +54,18 @@ For more detailed, step-by-step guides to setup and run the IoT Hub certificate
 - Espressif ESP32: [How to Setup and Run Azure SDK for Embedded C IoT Hub Client on ESP32](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/aziot_esp32/readme.md)
 
 - Azure Device Update with Espressif ESP32 on Arduino IDE: [How to Setup and Run Azure SDK for Embedded C ADU on Espressif ESP32](https://github.com/Azure/azure-sdk-for-c-arduino/blob/main/examples/Azure_IoT_Adu_ESP32/readme.md)
-
 - VxWorks: [How to Setup and Run Azure SDK for Embedded C IoT Hub Client and Provisioning Client Samples on VxWorks](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_vxworks.md)
 
+### Community Samples
+
+Beyond the samples above, created, maintained and supported by Microsoft, there are additional samples created and maintained by the community:
+
+- Raspberry Pi Pico: [Azure IoT Hub RPI Pico LDR with Cloud to Device Methods](https://github.com/djaus2/Azure_IoT_Hub_Arduino_RPI_Pico_Telemetry/tree/master/Azure_IoT_Hub_RPI_Pico_LightSensor_WithCDM) created and maitained by [David Jones](https://github.com/djaus2), Microsoft MVP Windows Embedded and IoT App Developer.
+
```

Comment:
also add this?
https://github.com/microsoft/azure-iot-central-esp32-sample

Reply:
not sure if the timing for this one is right.... let's talk during stand-up.

Reply:
resolving for now, but glad to add this later if we decide to do so.